### PR TITLE
Do not replay common init

### PR DESCRIPTION
**Pull request by duck-rh**

### CODE REVIEW

File: tasks/common.yml [hunk 1]
```diff
@@ -52,3 +52,8 @@
     line: "rotate {{ log_retention_week }}"
     state: present
   when: log_retention_week is defined
+
+- name: Set http_common
+  set_fact:
+    httpd_common: True
```
Comment (mscherer) [inline tasks/common.yml]: I would rather use a variable prefixed by _, to denote that's internal. Current code base is a bit inconsistant with that, but I rather not augment entropy.

**Review by mscherer — CHANGES_REQUESTED** [review]
Also, please merge the 2 PR in 1. I spent 5 minutes to try to figure the order of the PR and where was the first one.

**Comment by mscherer** [comment]
Ok, I mean the 2 commits, sorry

**Comment by duck-rh** [comment]
Let me know if you see more improvements you would like

**Comment by mscherer** [comment]
So I would be ok to merge this, but I now realize that I do not understand the problem it is supposed to fix. 
(also, I think maybe ansible upstreal should support something to include a file only once, like it does for the dependencies )

**Review by mscherer — APPROVED** [review]

**Comment by duck-rh** [comment]
I just extended whet you already started yourself. The goal is to avoid repeating rules we are sure were applied and make it faster.

Upstream support would be nice but sometimes you really want to replay them because you know something has changed, like when you want to change httpd server-level config as suggested in #36

**Comment by duck-rh** [comment]
I'm gonna merge and we can re-discuss this later. At the moment it follows the role's logic and works.

**Pull request merged**